resources/views/world/_feature_entry.blade.php

```diff
@@ -16,14 +16,14 @@
         <div class="world-entry-text parsed-text">
             {!! $feature->parsed_description !!}
         </div>
-        @if($feature->altTypes->count())
+        @if($feature->altTypesVisible->count())
             <h5 class="inventory-header">
                 Alternate Types
                 <a class="small collapse-toggle collapsed" href="#alt-{{ $feature->id }}" data-toggle="collapse">Show</a></h3>
             </h5>
             <div class="collapse show" id="alt-{{ $feature->id }}">
                 <ul>
-                    @foreach($feature->altTypes as $altType)
+                    @foreach($feature->altTypesVisible as $altType)
```

Comment:
Likewise this becomes `$feature->altTypes()->visible()->get()`.

Reply:
Edited, tested- worked!